### PR TITLE
fix: Lemonize all funnel filters

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- lemonizes all filters except "chart type" which requires a custom lemon component

## Changes

Screen Shot 2022-09-08 at 2 06 38 PM
Screen Shot 2022-09-09 at 2 07 38 PM
Screen Shot 2022-09-09 at 2 08 39 PM

Screen Shot 2022-09-09 at 11 21 07 AM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by clarkus — CHANGES_REQUESTED** [review]
There are updated icons for the funnel ordering. Those are available here:

- [Left to right](https://www.figma.com/file/Y9G24U4r04nEjIDGIEGuKI/PostHog-Design-System-One?node-id=3139%3A4638)
- [Top to bottom](https://www.figma.com/file/Y9G24U4r04nEjIDGIEGuKI/PostHog-Design-System-One?node-id=3977%3A2036)

funnel-ltr
funnel-ttb

**Review by benjackwhite — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/insights/filters/AggregationSelect.tsx [hunk 1]
```diff
@@ -13,33 +13,57 @@ interface AggregationSelectProps {
...
+
+    groupTypes.forEach((groupType) => {
+        options.push({
+            value: groupType.group_type_index,
+            label: `Unique ${aggregationLabel(groupType.group_type_index).plural}`,
+        })
+    })
+
+    if (
+        [GroupsAccessStatus.HasAccess, GroupsAccessStatus.HasGroupTypes, GroupsAccessStatus.NoAccess].includes(
```
[the remark this answers is not in the document]
Comment (EDsCODE) [reply frontend/src/scenes/insights/filters/AggregationSelect.tsx:37]: yeah, replied to other comment. HasAccess doesn't mean it's used yet. `AlreadyUsing` is the enabled state which is basically HasAccess + HasGroupTypes together

**Review by benjackwhite — APPROVED** [review]
More of a quick look but all seems good to me :shipit:

**Review by clarkus — CHANGES_REQUESTED** [review]
Generally looks right to me. We should use sentence-case for all the menu item labels. Right now we have a mix of title-case and sentence. https://posthog.com/handbook/company/communication

**Comment by EDsCODE** [comment]
Should be updated now @clarkus

**Review by clarkus — APPROVED** [review]

**Pull request merged**